src/ssl/ssl_openssl_impl.cpp

```diff
@@ -620,9 +620,11 @@ void free(void* ptr, const char* file, int line) { Memory::free(ptr); }
 void OpenSslContextFactory::internal_init() {
   CRYPTO_set_mem_functions(openssl::malloc, openssl::realloc, openssl::free);
 
+ #if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
Maybe move these init. functions into the #ifdef block below (starting at line 629)?

Reply:
Maybe not. I don't think this will work for libressl.